scripts/test_fundraising/sale.py

```diff
@@ -0,0 +1,62 @@
+import json
+import subprocess
+import datetime
+import time
+
+valAddress = 'spnvaloper1ezptsm3npn54qx9vvpah4nymre59ykr9mx22g4'
+saleTemplateFile = './sale_template.json'
+saleFile = './sale.json'
+
+def cmd(command):
+    subprocess.run([command], shell=True, check=True)
+
+def cmdDevnull(command):
+    subprocess.run([command], shell=True, check=True, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+
+def datef(d):
+    return d.isoformat("T") + "Z"
+
+def setSaleJSON(sellingDenom, sellingAmount, payingDenom, price, startTime, endTime):
+    f = open(saleTemplateFile)
+    jf = json.load(f)
+    jf['selling_coin']['denom'] = sellingDenom
+    jf['selling_coin']['amount'] = sellingAmount
+    jf['paying_coin_denom'] = payingDenom
+    jf['start_price'] = price
+    jf['start_time'] = startTime
+    jf['end_time'] = endTime
+    with open(saleFile, 'w', encoding='utf-8') as newF:
+        json.dump(jf, newF, ensure_ascii=False, indent=4)
+
+# Initialization
```

Comment:
Although probably not important here, I'd always advise including running code directly in the script and instead rely on the [main boilerplate](https://www.python-boilerplate.com/py3+executable)

Reply:
Yes, it's just because it's simpler to modify without another indentation